crates/nu-command/tests/commands/which.rs

```diff
@@ -21,6 +21,16 @@ fn which_alias_ls() {
     assert_eq!(actual.out, "Nushell alias: ls -a");
 }
 
+#[test]
+fn which_custom_alias() {
+    let actual = nu!(
+        cwd: ".", 
+        r#"alias foo = print "foo!"; which foo | get path.0 | str trim"#
```

Comment:
Here you can omit the `cwd: "."` as it not necessary for the test to access the file system.

I would suggest making the test a bit more broad by changing the output to `| to nuon --raw` instead of just testing one field. Then we can also validate the `built-in` field in the same test.

Reply:
done!